packages/market-service/sample.env

```diff
@@ -0,0 +1,4 @@
+COINGECKO_API_KEY='CG-we5Z4KbdYMCgMUVAqDjQMWfc'
+ETHEREUM_NODE_URL='https://dev-daemon.ethereum.shapeshift.com'
+UNCHAINED_ETHEREUM_HTTP_URL='https://dev-api.ethereum.shapeshift.com'
+UNCHAINED_ETHEREUM_WS_URL='wss://dev-api.ethereum.shapeshift.com'
```

Comment:
Do we need these for this PR? These are stemming from old implementations, if you look at e.g types of `FoxyMarketService` instantiation, you will notice `providerUrls` which we pass from web, which is now how we pass env vars:

https://github.com/shapeshift/lib/blob/505516c23b582ff1cd2d684b6c84a117157e3423/packages/market-service/src/foxy/foxy.ts#L21